libs/accounts/recovery-phone/src/lib/recovery-phone.service.spec.ts

```diff
@@ -224,6 +234,35 @@ describe('RecoveryPhoneService', () => {
       expect(mockRecoveryPhoneManager.getUnconfirmed).toBeCalledWith(uid, code);
     });
 
+    it('can handled simPumpingRiskScore throwing PhoneNumberNotSupported error', async () => {
+      const phoneNumber = '+15005550000';
+      const smsPumpingRisk = 95;
+
+      mockRecoveryPhoneManager.getUnconfirmed.mockReturnValueOnce({
+        isSetup: true,
+        phoneNumber,
+      });
+      mockSmsManager.phoneNumberLookup.mockReturnValueOnce({
+        phoneNumber,
+        smsPumpingRisk,
+      });
+
+      const promise = service.confirmSetupCode(uid, code);
+      await expect(promise).rejects.toThrow(/Phone number not supported.*/);
```

Comment:
I can see how we might want to review the error shown to users here, though I suppose there isn't really anything a user can do if they're incorrectly flagged.

Reply:
I think it's better not to divulge too much info in our errors. IMO, we should trust the score twilio provides and not worry about falsely identifying users. We have metrics and logs as well to see trends and understand its impact.